pkg/ingester/activeseries/active_series.go

```diff
@@ -118,31 +124,45 @@ func (c *ActiveSeries) ContainsRef(ref uint64) bool {
 	return c.stripes[stripeID].containsRef(ref)
 }
 
-// Active returns the total number of active series. This method does not purge
+// Active returns the total number of active series and the total number
+// of active native histogram series. This method does not purge
 // expired entries, so Purge should be called periodically.
-func (c *ActiveSeries) Active() int {
+func (c *ActiveSeries) Active() (int, int, int) {
 	total := 0
+	totalNativeHistograms := 0
+	totalNativeHistogramBuckets := 0
 	for s := 0; s < numStripes; s++ {
-		total += c.stripes[s].getTotal()
+		all, histograms, buckets := c.stripes[s].getTotal()
+		total += all
+		totalNativeHistograms += histograms
+		totalNativeHistogramBuckets += buckets
 	}
-	return total
+	return total, totalNativeHistograms, totalNativeHistogramBuckets
 }
 
 // ActiveWithMatchers returns the total number of active series, as well as a
 // slice of active series matching each one of the custom trackers provided (in
-// the same order as custom trackers are defined). This method does not purge
+// the same order as custom trackers are defined), and then the same thing for
+// only active series that are native histograms. This method does not purge
 // expired entries, so Purge should be called periodically.
-func (c *ActiveSeries) ActiveWithMatchers() (int, []int) {
+func (c *ActiveSeries) ActiveWithMatchers() (int, []int, int, []int, int, []int) {
```

Comment:
Similar to above, we could name the return values. This way you don't have to initialize them , nor do you have to list them in `return`, just use bare `return`

```suggestion
func (c *ActiveSeries) ActiveWithMatchers() (total int, totalMatching []int, totalNativeHistograms int, totalMatchingNativeHistograms []int, totalNativeHistogramBuckets int, totalNativeHistogramBuckets []int) {
```